README.md

```diff
@@ -1409,6 +1409,8 @@ import type { DroppableProvided } from 'react-beautiful-dnd';
 
 If you are using [TypeScript](https://www.typescriptlang.org/) you can use the community maintained [DefinitelyTyped type definitions](https://www.npmjs.com/package/@types/react-beautiful-dnd). [Installation instructions](http://definitelytyped.org/).
 
+Here is an [example written in typescript](https://github.com/abeaudoin2013/react-beautiful-dnd-multi-list-typescript-example).
```

Comment:
*Here is

Reply:
It looks like it is just an issue with the diff view

Reply:
The rich diff is correct:
https://github.com/atlassian/react-beautiful-dnd/pull/666/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8